CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+<!--
+  Copyright 2019 Google LLC
```

Comment:
The copyright year should be the file was first published.

Reply:
Ah, thanks for clarifying. I'll go ahead and change this on my other PRs as well